onnxruntime/python/tools/tensorrt/perf/build/Dockerfile.tensorrt-perf

```diff
@@ -5,7 +5,9 @@
 # Dockerfile to run ONNXRuntime with TensorRT integration
 
 # nvidia TensorRT Base Image
-FROM nvcr.io/nvidia/tensorrt:20.12-py3
+ARG TRT=21.04
```

Comment:
same comment here. This is almost a duplicate of https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/Dockerfile.tensorrt
let's try to consolidate as much as possible.